Displays.3dshapes/CREDITS.md

```diff
@@ -3,23 +3,25 @@
 The credits of 3D models are listed in respective folder.  
 Please fill all 3D models credits and info source data accordingly.  
 
-
 ### Info requirements:
 - Author
 - if the 3D model is script generated, a link to the script generator is required as well as the version and sw used
 - if the 3D model is manually generated, include the source model @ https://github.com/KiCad/packages3D-source
 
-<hr>  
-
-### Displays.3dshapes:  
+<hr>
 
+#### AG12864E
+STEP and WRL models designed by Stefan Olsson
+Author:	Stefan Olsson
+Tool:	- FreeCAD 0.17 Build 11223 64 bit Microsoft windwos platform https://www.freecadweb.org/
 
+### Displays.3dshapes:  
 HDSP-4830, HDSP-4832, HDSP-4836, HDSP-4840, HDSP-4850, HDSP-48xx, HLCP-J100
```

Comment:
This line should start with #### (I seem to have missed it when i merged the credits file the first time round.)